### PR TITLE
chore(userconfigs): switch to go-api-schemas

### PR DESCRIPTION
**Pull request by byashimov**
To get updates by dependabot.

### CODE REVIEW

File: charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml [hunk 1]
```diff
@@ -637,6 +637,7 @@ spec:
                     - "13"
                     - "14"
                     - "15"
+                    - "10"
```
Comment (Serpentiel) [inline charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml:640]: this has been [deprecated](https://github.com/aiven/go-api-schemas/blob/main/pkg/dist/service_types.yml#L4290-L4292) 🤔

**Review by Serpentiel — CHANGES_REQUESTED** [review]

File: charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml (hunk 1 above)
Comment (byashimov) [reply charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml:640]: So thought I, but then checked out the source json: 
image

Comment (byashimov) [reply charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml:640]: Please double check this. It looks weird.

Comment (byashimov) [reply charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml:640]: image
It is available through the console. And also come out too.

Comment (Serpentiel) [reply charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml:640]: "coming out of support" means it's deprecated

Comment (Serpentiel) [reply charts/aiven-operator-crds/templates/aiven.io_postgresqls.yaml:640]: there is no v10 in the source JSON:

```json
{
   "pg_version":{
      "title":"PostgreSQL major version",
      "type":[
         "string",
         "null"
      ],
      "enum":[
         "11",
         "12",
         "13",
         "14",
         "15"
      ]
   }
}
```

**Comment by byashimov** [comment]
Closing this as the package has been damaged by invalid schema.

**Comment by Serpentiel** [comment]
duplicates #327

**Pull request closed without merging**